sources/pipedrive/helpers/__init__.py

```diff
@@ -5,18 +5,6 @@
 from itertools import groupby
 
 
-def parse_timestamp(timestamp: Any) -> str:
-    if not timestamp:
-        return None
-    if isinstance(timestamp, str):
-        timestamp = pendulum.parse(timestamp)
-    assert isinstance(
-        timestamp, pendulum.DateTime
-    ), "since_timestamp must be a valid ISO datetime string or pendulum.DateTime object"
-    timestamp = timestamp.in_timezone("UTC")
-    return cast(str, timestamp.to_iso8601_string().replace("T", " ").replace("Z", ""))
```

Comment:
It's used only for Pipedrive, so it's no longer needed

Reply:
thanks for cleaning that up!